ingest.py

```diff
@@ -1,316 +0,0 @@
-            header = self._create_header(header_add)
-            if regular_table:
-                self._table_to_db('regular', regular_table, header)
-            if playoff_table:
-                self._table_to_db('playoff', playoff_table, header)
-
-    @staticmethod
-    def _get_header_add(table):
-        """Finds and returns the header of a table.
-
-        :param table: A basketball-reference stats table.
-        :returns: Header of the table as a list of strings.
-        """
-        def replace_titles(title):
-            return multiple_replace(
-                title,
-                {'%': 'P', '3': 'T', '+/-': 'PlusMinus'})
-        titles = (
-            replace_titles(str(th.get_text()))
-            for th in table.find_all('th')
-        )
-        return list(unique_everseen(titles))
-
-    def _create_header(self, header_add):
-        """
-        Creates the header including the initial header and the header_add.
-
-        :param header_add: List of strings found from the table to add to the
-            initial_header.
-        """
-        header = self._initial_header + header_add
-        header[9] = 'Home'
-        header.insert(11, 'WinLoss')
-        return without(header, 'FGP', 'FTP', 'TPP')
-
-    def _table_to_db(self, season, table, header):
-        """
-        Adds all gamelogs in a table to the database.
-
-        :param season: The season type of the gamelog, either 'playoff' or
-            'regular'.
-        :param table: A basketball-reference stats table.
-        :param header: Header of the table.
-        """
-        gamelogs = []
-        for row in table.find_all('tr'):
-            stat_values_add = self._stat_values_parser(row.find_all('td'))
-            # Skip headers which are found throughout the table.
-            if not stat_values_add:
-                continue
-            stat_values = (
-                self._initial_stat_values +
-                [season] +
-                stat_values_add
-            )
-            gamelogs.append(dict(izip(header, stat_values)))
-        self._insert_gamelogs(gamelogs)
-
-    @staticmethod
-    def is_number(s):
-        """
-        Checks if a string is a number.
-
-        :param s: String to check whether is a number or not.
-        :returns: A boolean of whether or not its a number.
-        """
-        try:
-            float(s)
-            return True
-        except ValueError:
-            return False
-
-    def _stat_values_parser(self, cols):
-        """
-        Returns a list of values which change or skip the col strings based
-        on their content.
-
-        :param cols: Each column being a stat value in a row.
-        :returns: The stat values after they are parsed.
-        """
-        def get_val(i, col):
-            text = str(col.get_text())
-            conversion = self._conversions.get(i)
-            if conversion:
-                return conversion(text)
-            if self.is_number(text):
```

Comment:
Why don't you make a number_conversion method instead of breaking this out into it's own case? Are there too many numbers?